### DISCUSSION TITLE
Copy URL to SN

### DISCUSSION THREAD

**Discussion by CyberSmurfs** [standardnotes/forum#3360] [Suggestions]
Hi

Does Standard notes have a feature where you can copy the URL from an external website and then past it in Standard Notes...if not I suggest that SN have a feature like that?

**Comment by effieeee** [comment]
Hmm, can you elaborate a bit on this? Currently when you copy a URL and paste it into either the **Super** or **Rich Text** note types, it converts into a clickable hyperlink. Would this be what you were looking for?

**Reply by CyberSmurfar** [reply]
Yes tats what I am looking for.
